### ISSUE TITLE
[MSFT-181] Form elements must have labels

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2851]
**Steps to Reproduce**
1. Navigate to https://accounts.topcoder.com/member
2. Right click > Inspect > Elements
3. Check input elements

**Expected Result**
The respective elements should at least have:
- aria-label attribute
- aria-labelled by attribute
- an implicit (wrapped) <label>
- an explicit <label>
- title attribute

**Actual Result**
The following input elements have missing labels

<input ng-model="vm.username" name="username" placeholder="Username or Email" type="text" required="" class="ng-pristine ng-untouched ng-empty ng-invalid ng-invalid-required" _vkenabled="true">

<input id="current-password-input" ng-model="vm.currentPassword" ng-model-options="{allowInvalid: true}" ng-focus="vm.onCPFocus($event)" ng-blur="vm.onCPBlur($event)" name="currentPassword" type="password" placeholder="Password" required="" class="input input-btn ng-pristine ng-untouched ng-empty ng-invalid ng-invalid-required" _vkenabled="true">

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 3. Understandable
**WCAG Level**: A
**WCAG Success Criterion**: 3.3.2 Labels or Instructions
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=37e62274%2D78ab%2D496a%2D918f%2Dabba8f38f3fe&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fyoctocode%2Fbug%203

**Comment by mishacucicea** [comment]
The fields have placeholders, which are getting read, so not sure if we need to change the approach. @Oanh-and-only-Oanh?

**Comment by crazyk07** [comment]
https://github.com/topcoder-platform/community-app/issues/2763 Covered by this.
